From 1da7cdeec3823bc98914330fca96cd5d93538d51 Mon Sep 17 00:00:00 2001
From: Loc Pham <loc.pham.xh@rvc.renesas.com>
Date: Fri, 21 Jul 2017 17:18:36 +0700
Subject: [PATCH] ARM: DTS: iwg20m: enable XHCI

XHCI is enable for suporting USB3.0
in RZG1 hardware manual, USB3.0 use one USB2.0 PHY, thereforce, USB1 need to be
disabled in order to enable USB3.0

Signed-off-by: Loc Pham <loc.pham.xh@rvc.renesas.com>
---
 arch/arm/boot/dts/r8a7743-iwg20m.dts | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7743-iwg20m.dts b/arch/arm/boot/dts/r8a7743-iwg20m.dts
index 6609d23..4efbd68 100644
--- a/arch/arm/boot/dts/r8a7743-iwg20m.dts
+++ b/arch/arm/boot/dts/r8a7743-iwg20m.dts
@@ -359,15 +359,13 @@
 	status = "okay";
 };
 
-&pci0 {
+&xhci {
 	status = "okay";
-	pinctrl-0 = <&usb0_pins>;
-	pinctrl-names = "default";
 };
 
-&pci1 {
+&pci0 {
 	status = "okay";
-	pinctrl-0 = <&usb1_pins>;
+	pinctrl-0 = <&usb0_pins>;
 	pinctrl-names = "default";
 };
 
-- 
2.7.4

